src/Tests/ExecutionPathTracerTests.cs

```diff
@@ -533,4 +557,71 @@ public void BigTest()
             Assert.AreEqual(expected.ToJson(), path.ToJson());
         }
     }
+
+    [TestClass]
+    public class MeasurementTests : ExecutionPathTracerTests
```

Comment:
This is just a reordering of existing tests from lower down in this file (no new changes included here).